diff --git a/index.d.ts b/index.d.ts
index 53dc74d77277c7fa4fd459caa7ed261bf3a0ea02..71532cbd99dca6bcf102c000345156d39950a5b0 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -21,6 +21,8 @@ import * as serveStatic from 'serve-static';
 import * as core from 'express-serve-static-core';
 import * as qs from 'qs';
 
+export { Express } from 'express-serve-static-core';
+
 /**
  * Creates an Express application. The express() function is a top-level function exported by the express module.
  */